config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :referral_agencies, :landlords
```

Comment:
we don't have things like index so could you add an "except" at the end to take away the methods that are included in resources but not here (you could also use the only statement)

https://stackoverflow.com/questions/6462815/except-options-in-routes-that-contain-collections-and-members-not-working